be/src/Project/Services/Client/Client.Api/Infrastructure/Clients/Entities/ClientMarketingApproval.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Foundation.GenericRepository.Entities;
+using System;
+
+namespace Client.Api.Infrastructure.Clients.Entities
+{
+    public class ClientMarketingApproval : Entity
```

Comment:
**suggestion** Add initial seed for entity. You can see example in News.Api. Seeds directory